src/data/followings.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "_id": "123",
+    "username": "guyfieri",
+    "firstName": "Guy",
+    "lastName": "Fieri",
+    "type": "REG USER",
+    "avatar": "guy.jpg"
+  },
+  {
+    "_id": "345",
+    "username": "gr123",
+    "firstName": "Gordon",
+    "lastName": "Ramseya;sldjfa;sidjf",
```

Comment:
Hi Macee, not sure whether you created a long last name on purpose to show what the page will be like. Ramsey repeated three times here.

Reply:
Oh yes, I intentionally wrote it a few times just to show what it'd look like if someone's name is long haha (will wrap around instead of sticking out of its component)

Reply:
Sounds good!